Update which-sdk.mdx (#226)
hashicorp/terraform-docs-common: website/docs/plugin/which-sdk.mdx

There are two SDKs that you can use to build Terraform providers:

> * [SDKv2](/plugin/sdkv2) is what the majority of existing providers are built on, and it continues to provide a stable development experience.

Reviewer: ```suggestion
* [SDKv2](/plugin/sdkv2) is the foundation for the majority of existing providers and continues to provide a stable development experience.
```

Suggest editing for clarity and to use fewer words

* [Terraform Plugin Framework](/plugin/framework) is a new SDK, realigning the provider development experience and abstractions around the updated architecture of Terraform. It represents our vision of the future of Terraform plugin development.

> This guide will help you decide whether you should continue using SDKv2, or begin using the framework.

Reviewer: ```suggestion
This guide helps you decide whether you should continue using SDKv2 or begin using the framework.
```

Suggest editing to avoid a future tense style issue

## Are You Writing a New Provider or Maintaining an Existing Provider?

If you maintain a large existing provider, we recommend that you begin migrating from SDKv2 to the framework by developing new resources and data sources with the framework. You can use [terraform-plugin-mux](/plugin/mux) to refactor the provider over time. Refer to the [Migrate to Terraform Plugin Framework](/plugin/sdkv2#migrate-to-terraform-plugin-framework) documentation for more details.

> If you are developing a new provider, we recomment you to building it entirely with the framework.

Reviewer 2: ```suggestion
If you are developing a new provider, we recommend you to building it entirely with the framework.
```

> If you are developing a new provider, we recommend you to building it entirely with the framework.

Reviewer 3: ```suggestion
If you are developing a new provider, we recommend you build it entirely with the framework.
```
